students/hatims/Session01/break_me.py

```diff
@@ -0,0 +1,21 @@
+def Name_Error():
+    """This function will raise a NameError exception when called """
+    print(num)
```

Comment:
You should actually fill these in with built-in errors for each type. See [this page](https://docs.python.org/3/library/exceptions.html) for more information.